### PR TITLE
Disabled Accordion in Dashboard

### PR DESCRIPTION
**Pull request by cheina97**
Components updated:

- TemplatesTable

This PR disable the accordion in the Dashboard page.

### CODE REVIEW

File: frontend/src/components/workspaces/Templates/TemplatesTable/TemplatesTable.tsx [hunk 1]
```diff
@@ -91,34 +91,29 @@ const TemplatesTable: FC<ITemplatesTableProps> = ({ ...props }) => {
...
+        ? setExpandedId(old => (create ? old : old.filter(id => id !== rowId)))
+        : setExpandedId(old => [...old, `${rowId}`]);
     }
   };
 
   useEffect(() => {
-    window.sessionStorage.setItem('prevExpandedIdDashboard', `${expandedId}`);
+    window.sessionStorage.setItem(
+      'prevExpandedIdDashboard',
+      `${expandedId.toString()}`
```
Comment (QcFe) [inline frontend/src/components/workspaces/Templates/TemplatesTable/TemplatesTable.tsx:113]:
```suggestion
      expandedId.join(',')
```
Always better to keep parsing and marshalling symmetrical.

**Review by QcFe — APPROVED** [review]
Minor comment

**Comment by QcFe** [comment]
Check staging before going, though :)

**Comment by QcFe** [comment]
/hold
something is not working, the accordion is not expanding anymore

**Comment by QcFe** [comment]
Also couldn't we extend this behavior to the active view as well?

**Comment by cheina97** [comment]
> Also couldn't we extend this behavior to the active view as well?

Of course but it can be done in another PR for faster development.

**Comment by cheina97** [comment]
> Something is not working, the accordion is not expanding anymore

I've just tried and it works locally.

**Pull request merged**